### PR TITLE
Implement MBSeconds Metric on 1.5.X

### PR DESCRIPTION
**Pull request by PineappleIOnic**
## What does this PR do?

This PR implements the MBSeconds Metric on functions and builds in 1.5.x

## Test Plan

Update relevant tests that check metrics

## Checklist

- [X] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [ ] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

**Review by abnegate — COMMENTED** [review]
Functions tests are failing after a few retries

File: tests/e2e/Services/Functions/FunctionsConsoleClientTest.php [hunk 1]
```diff
@@ -92,23 +92,26 @@ public function testGetCollectionUsage(array $data)
...
         $this->assertIsNumeric($response['body']['executionsTotal']);
         $this->assertIsNumeric($response['body']['executionsTimeTotal']);
+        $this->assertIsNumeric($response['body']['executionsMbSecondsTotal']);
         $this->assertIsArray($response['body']['deployments']);
         $this->assertIsArray($response['body']['deploymentsStorage']);
         $this->assertIsArray($response['body']['builds']);
         $this->assertIsArray($response['body']['buildsTime']);
         $this->assertIsArray($response['body']['buildsStorage']);
-        $this->assertIsArray($response['body']['buildsTime']);
+        $this->assertIsArray($response['body']['buildsMbSeconds']);
```
[the remark this answers is not in the document]
Comment (PineappleIOnic) [reply tests/e2e/Services/Functions/FunctionsConsoleClientTest.php:111]: https://github.com/appwrite/appwrite/pull/8385/files/98ae495879456f2c9f02e962a1a83138e8076ecd#diff-aba8acb48a553bd56ea7621c504256ef623159b290ad0e2c279f68d2b24c9b48R108-R111

BuildsTime was already being tested so I just replaced the duplicate

File: src/Appwrite/Platform/Workers/Builds.php [hunk 2]
```diff
@@ -542,9 +542,11 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                 ->addMetric(METRIC_BUILDS, 1) // per project
                 ->addMetric(METRIC_BUILDS_STORAGE, $build->getAttribute('size', 0))
                 ->addMetric(METRIC_BUILDS_COMPUTE, (int)$build->getAttribute('duration', 0) * 1000)
+                ->addMetric(METRIC_BUILDS_MB_SECONDS, $function->getAttribute('memory') * $build->getAttribute('duration', 0))
```
Comment (abnegate) [inline src/Appwrite/Platform/Workers/Builds.php:545]: Is `duration` seconds or millis? I see some places we multiply by 1000

Comment (PineappleIOnic) [reply src/Appwrite/Platform/Workers/Builds.php:545]: It is stored in seconds: https://github.com/appwrite/appwrite/blob/cbea4264eaf72eb8bde43a2cd8eb24103904ad0c/src/Appwrite/Utopia/Response/Model/Build.php#L60-L65

**Review by abnegate — APPROVED** [review]

**Pull request merged**